infrastructure/loadtesting/terraform/readme.md

```diff
@@ -10,3 +10,8 @@ If you require changes beyond whats described here, contact @zwinnerman-fleetdm.
 ### Running migrations
 After applying terraform with the commands above:
 `aws ecs run-task --region us-east-2 --cluster fleet-backend --task-definition fleet-migrate:"$(terraform output -raw fleet_migration_revision)" --launch-type FARGATE --network-configuration "awsvpcConfiguration={subnets="$(terraform output -raw fleet_migration_subnets)",securityGroups="$(terraform output -raw fleet_migration_security_groups)"}"`
+
+### Running a loadtest
```

Comment:
@chiiph do you think this is readable?

Reply:
We need a branch to be pushed for this on the main repo. Currently Fleet does not have a way to make a local docker image easily, so I opted for pulling from ECR. Ideally we would build locally and push that to ECR though.

Reply:
I like the new wording.

Reply:
> We need a branch to be pushed for this on the main repo.

Ok, then I would explain this as well at the beginning.